tlsn/examples/src/arg_parse.rs

```diff
@@ -0,0 +1,60 @@
+use std::collections::VecDeque;
```

Comment:
i use https://docs.rs/structopt/latest/structopt/ for CLI parsing, aha just suggesting in case you find it better, but feel free to stick to this :)